io.github.yuxshao.ptcollab.yml

```diff
@@ -0,0 +1,26 @@
+id: io.github.yuxshao.ptcollab
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: ptcollab
+rename-icon: ptcollab
+rename-desktop-file: ptcollab.desktop
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --device=dri
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --socket=fallback-x11
+  - --filesystem=home
+modules:
+  - name: ptcollab
+    buildsystem: qmake
+    sources:
+      - type: git
+        url: https://github.com/yuxshao/ptcollab.git
+        tag: "v0.6.4.8"
+#        commit: "5feb27decb9e0600d1defdaf9dc1424c6fdcb7ad"
+    build-commands:
+#      - mkdir /app/share/metainfo -p
+#      - cp io.github.yuxshao.ptcollab.metainfo.xml /app/share/metainfo/
```

Comment:
why did you ignore the suggestion I made for this?

Reply:
what

Reply:
sorry sorry sorry

Reply:
i saw it

Reply:
I removed that line now, it is running fine!